.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
```

Comment:
why restrict this? let's run on all branches - I can't see it running now, and I need to 

for comparison https://github.com/OpenUpSA/muni-portal-backend/blob/master/.github/workflows/test.yml#L4